### DISCUSSION TITLE
TimeInput custom ranges for hours and minutes

### DISCUSSION THREAD

**Discussion by atumas-bananamilk** [mantinedev/mantine#5949] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

^7.2.2

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

None

### Describe the bug

Is it possible to set custom ranges of hours and minutes in TimeInput component?
So instead of all options for the minute be 0 - 60, it should just be 0 / 15 / 30 / 45 (4 options)?

![Screenshot 2024-03-18 at 20 39 51](https://github.com/mantinedev/mantine/assets/22228431/430b9a47-c636-4f04-870c-5f2812fae44a)

### Self-service

- [ ] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
`TimeInput` is a regular `input[type=time]`, it supports all props of the `input` element and does not support any extra features
